packages/logger/src/lib/logger.ts

```diff
@@ -3,14 +3,14 @@ import { hashMessage } from 'ethers/lib/utils';
 import { version } from '@lit-protocol/constants';
 
 export enum LogLevel {
-  INFO = 0,
-  DEBUG = 1,
-  WARN = 2,
-  ERROR = 3,
+  OFF = -1,
+  ERROR = 0,
+  INFO = 1,
+  DEBUG = 2,
```

Comment:
Note: These orderings are not standard. going to come back and re work the level enumeration weights in a fast follow PR with other changes we discussed.

Reply:
Maybe use the levels from another lib like [ethers](https://docs.ethers.org/v5/api/utils/logger/#Logger-levels)?